scripts/build/build.mk

```diff
@@ -34,9 +34,9 @@ deps:
 	$(GO_GET) -v -t -d ./...
 
 .PHONY: $(PLATFORMS)
-$(PLATFORMS): deps
+$(PLATFORMS): deps generate
 	mkdir -p $(OUT_DIR)
-	GOOS=$(os) GOARCH=$(ARCH) $(GO_BUILD) $(GO_BUILD_FLAGS) -ldflags "$(GO_BUILD_LDFLAGS)" -o $(OUT_DIR)/$(BINARY)-$(VERSION)-$(os)-$(ARCH) ./cmd
+	CGO_ENABLED=0 GOOS=$(os) GOARCH=$(ARCH) $(GO_BUILD) $(GO_BUILD_FLAGS) -ldflags "$(GO_BUILD_LDFLAGS)" -o $(OUT_DIR)/$(BINARY)-$(VERSION)-$(os)-$(ARCH) ./cmd
```

Comment:
After Adding the `CGO_ENABLED` flag, does it work?

Reply:
> After Adding the `CGO_ENABLED` flag, does it work?

The result meets expectations, and it will indeed be a bit more convenient.

Reply:
Then, should we delete the ENV declare under the https://github.com/apache/skywalking-rover/tree/main/docker?

Reply:
> Then, should we delete the ENV declare under the https://github.com/apache/skywalking-rover/tree/main/docker?

You are right. ^_^

Reply:
Please also remove this ENV in the https://github.com/apache/skywalking-rover/blob/main/docker/Dockerfile.debug.

Reply:
> Please also remove this ENV in the https://github.com/apache/skywalking-rover/blob/main/docker/Dockerfile.debug.

OK.